server/utils/utils.ts

```diff
@@ -124,9 +122,19 @@ const jsonDtTo12HourTime = (dt: string): string => {
   return momentTime.isValid() ? momentTime.format('hh:mm a') : null
 }
 
-const toDate = (date: string) => {
-  const [day, month, year] = date.split('/')
-  return new Date(`${year}-${month}-${day}`)
+const parseIsoDate = (date: string) => {
+  return date ? parse(date, 'yyyy-MM-dd', new Date()) : null
+}
+
+const parseCvlDate = (date: string) => {
+  return date ? parse(date, 'dd/MM/yyyy', new Date()) : null
+}
+
+const parseCvlDateTime = (date: string, { withSeconds }: { withSeconds: boolean }) => {
```

Comment:
specifically not added a default for `withSeconds` - probably safer to always have to specify and have to think/check about what is safe to use